php/libraries/Database.class.inc

```diff
@@ -271,7 +271,11 @@ class Database
     {
         $retVal = array();
         foreach ($set as $key => $val) {
-            $retVal[$key] = is_null($val) ? null : htmlspecialchars($val);
+            if ($val === 'Unrated') {
```

Comment:
@MounaSafiHarab I wanna venture a guess that this will throw PHP warvings in the error_log...

Could you test that quickly ?